cognite/neat/_graph/extractors/_classic_cdf/_classic.py

```diff
@@ -122,11 +134,15 @@ def extract(self) -> Iterable[Triple]:
             yield from self._extract_labels()
         except CogniteAPIError as e:
             warnings.warn(CDFAuthWarning("extract labels", str(e)), stacklevel=2)
+        else:
+            self._extracted_labels = True
```

Comment:
I see that self._extracted_labels is set to True, meaning `_get_activity_names` should never be called before `self.extract` is called?

Reply:
That is correct.